core/src/main/java/hivemall/fm/FactorizationMachineModel.java

```diff
@@ -92,6 +92,14 @@ protected float getW(int i) {
 
     protected abstract void setW(@Nonnull Feature x, float nextWi);
 
+    protected void setW(int i, float nextWi) {
```

Comment:
No need to have `protected void setW(int i, float nextWi)` and `protected void setW(@Nonnull String j, float nextWi)` in FactorizationMachineModel.

Reply:
`setW(int i, float nextWi)` is no more used when avoid caching in early stopping.

Reply:
ad24f38103b3951c6266382c048e0f4514dacd1e